docs/source/installation.rst

```diff
@@ -79,7 +85,7 @@ As an alternative, LinchPin can be installed via github. This may be done in ord
 
     (linchpin) $ git clone git://github.com/CentOS-PaaS-SIG/linch-pin
```

Comment:
This should now be `linchpin` as we've changed the name of the repo. Even though it will redirect, it would be better if we updated it here.